src/main/resources/io/fabric8/elasticsearch/plugin/kibana/applications_mapping.json

```diff
@@ -0,0 +1,6 @@
+{
+   "title":"$TITLE$",
+   "timeFieldName":"time",
```

Comment:
If we are providing the complete document there is no need to use what is provided in the message arg.